client/landing/subscriptions/comment-list/comment-row.tsx

```diff
@@ -0,0 +1,51 @@
+import { Gridicon } from '@automattic/components';
+import moment from 'moment';
+import { useMemo } from 'react';
+import type { PostSubscription } from '@automattic/data-stores/src/reader/types';
+
+const CommentRow = ( {
+	title,
+	excerpt,
+	site_title,
+	site_icon,
+	site_url,
+	date_subscribed,
+}: PostSubscription ) => {
+	const since = useMemo(
+		() => moment( date_subscribed ).format( 'LL' ),
+		[ date_subscribed, moment ]
+	);
+	const hostname = useMemo( () => new URL( site_url ).hostname, [ site_url ] );
+	const siteIcon = useMemo( () => {
+		if ( site_icon ) {
+			return <img className="icon" src={ site_icon } alt={ site_title } />;
+		}
+		return <Gridicon className="icon" icon="globe" size={ 48 } />;
+	}, [ site_icon, site_title ] );
```

Comment:
These can be extracted out in separate task - as currently there is a duplication of code between site-list and comment-list components.